core/src/css/component/decktracker/main/decktracker-deck-recap.component.scss

```diff
@@ -48,13 +48,14 @@
 		display: flex;
 		flex-direction: column;
 		justify-content: flex-start;
+		width: 150%;
```

Comment:
This is to make the deck title block wider. Without it, it would look like this.

![2022-09-06_12-53-56](https://user-images.githubusercontent.com/43519401/188605645-663ee0ed-d1c6-476f-8aca-b289fcf97f65.png)

Reply:
I think I understand what the issue is:
- The "deck-summary" element takes the full width
- It's divided between deck-image, deck-title and best-against
- deck-image has a fixed width, best-against is set to take 100% of the space

This means that the layout itself has to figure out what's the best way to give room to each widget.
There are multiple solutions:
- Attribute width to each element so that the total is 100% (you can have a mix of pixel width + flex-grow: 1)
- Keep the sizes dynamic, using flex-basis to tell the layout the relative size of the items

In this specific case, the component's width will always be constrained by the width of the ads column, so 400px. So I'm fine with using absolute pixel sizes in this case, but the other option is fine too.